examples/contrib/cifar10/main.py

```diff
@@ -52,8 +105,8 @@ def run(output_path, config):
 
     def train_step(engine, batch):
 
-        x = convert_tensor(batch[0], device=utils.device, non_blocking=True)
-        y = convert_tensor(batch[1], device=utils.device, non_blocking=True)
+        x = convert_tensor(batch[0], device=device, non_blocking=True)
```

Comment:
For TPU we can have an option to use ParallelLoader iterator, so maybe this wont apply...

Reply:
Ok. To check with TPU